static/app/utils/shouldUseLegacyRoute.tsx

```diff
@@ -0,0 +1,9 @@
+import {OrganizationSummary} from 'sentry/types';
+
+function shouldUseLegacyRoute(organization: OrganizationSummary) {
+  const {links} = organization;
+  const {organizationUrl} = links;
```

Comment:
```suggestion
  const {organizationUrl} = organization.links;
```

Do we need to destructure the property as we're not using the value.